### PR TITLE
Update protect-your-global-administrator-accounts.md

### PR DESCRIPTION
**Pull request by msbemba**
added note that a Global admin account can be created without any license. (For clarity)

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/737

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/protect-your-global-administrator-accounts.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ description: "Protect global administrator access to your Office 365 subscriptio
 
 # Protect your Office 365 global administrator accounts
 
+> [!NOTE]
+> The Global admin account can be created without any adding any license.
```
Comment (JohanFreelancer9) [inline Enterprise/protect-your-global-administrator-accounts.md]:
```suggestion
> A global administrator account can be created without adding any licenses.
```

**Comment by mypil** [comment]
@JoeDavies-MSFT @LauraWi - PR has been copy edited and is ready for final review. This is based on issue #737.

Thank you.

cc: @kenwith

**Pull request merged**